src/components/Course.vue

```diff
@@ -81,13 +82,13 @@ export default Vue.extend({
     };
   },
   computed: {
-    cautionString() {
+    cautionString(): string | null {
```

Comment:
Why not just string and if there is no caution it's just ''?

Reply:
That is very sketchy. I believe whether a variable exhibits useful information should be reflected by the type, instead of using a dummy value.